hack/istio/install-testing-demos.sh

```diff
@@ -179,7 +179,7 @@ EOF
     ${CLIENT_EXE} patch VirtualService bookinfo -n bookinfo --type json -p "[{\"op\": \"replace\", \"path\": \"/spec/hosts/0\", \"value\": \"${ISTIO_INGRESS_HOST}\"}]"
   fi
 
-  for namespace in bookinfo alpha beta
+  for namespace in bookinfo alpha beta gamma
```

Comment:
This seems relevant even for non-Maistra environments (waiting for the workloads in gamma seems unrelated to maistra).

Reply:
yes, you are right, I forgot on this minor change. this is not only for maistra, so this can also be backported into main